v3io/dataplane/response.py

```diff
@@ -67,7 +70,9 @@ def raise_for_status(self, expected_statuses=None):
         if (expected_statuses is None and self.status_code >= 300) or (
             expected_statuses and self.status_code not in expected_statuses
         ):
-            raise HttpResponseError("Request failed with status {0}: {1}".format(self.status_code, self.body))
+            raise HttpResponseError(
```

Comment:
Would be nice to add a test to `tests/test_client_errors.py`.

Reply:
done